modules/nw-ovn-ipsec-disable.adoc

```diff
@@ -23,24 +28,47 @@ $ oc patch networks.operator.openshift.io/cluster --type=json \
   -p='[{"op":"remove", "path":"/spec/defaultNetwork/ovnKubernetesConfig/ipsecConfig"}]'
 ----
 
-. Optional: You can increase the size of your cluster MTU by `46` bytes because there is no longer any overhead from the IPsec ESP header in IP packets.
+. To find the name of the OVN-Kubernetes data plane pod that exists on the `master` node in your cluster, enter the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ovn-kubernetes -l=app=ovnkube-master
+----
++
+.Example output
+[source,terminal]
+----
+ovnkube-master-5xqbf                      8/8     Running   0              28m
+...
+----
 
-. Verify that IPsec is disabled on your cluster:
+. Verify that the `master` node in your cluster has IPsec disabled by entering the following command. The command output must state `false` to indicate that the node has IPsec disabled.
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<XXXXX> \
-  ovn-nbctl --no-leader-only get nb_global . ipsec
+$ oc -n openshift-ovn-kubernetes -c nbdb rsh ovnkube-master-<pod_number_sequence> \// <1>
+  ovn-nbctl --no-leader-only get nb_global . ipsec 
 ----
+<1> Replace `<pod_number_sequence>` with the random sequence of letters, such as `5xqbf`, for the data plane pod from the previous step.
+
+. To remove the IPsec `ovn-ipsec` daemonset pod from the `openshift-ovn-kubernetes` namespace on the node, enter the following command:
 +
---
-where:
+[source,terminal]
+----
+$ oc delete daemonset ovn-ipsec -n openshift-ovn-kubernetes <1>
+----
+<1> The `ovn-ipsec` daemonset configures IPsec connections for east-west traffic on the node.
 
-`<XXXXX>`:: Specifies the random sequence of letters for a pod from the previous step.
---
+. Verify that the `ovn-ipsec` daemonset pod was removed from the `master` node in your cluster by entering the following command. If the command output does not list the pod, the removal operation is successful.
```

Comment:
The `ovn-ipsec` daemonset pod exists in all nodes, not only on the `master` node.

Reply:
:heavy_check_mark: